Draft version of NMP-2

pcardune opened the pull request:
@aldyen, @pcardune and @jaycenhorton sat down and put this together from @aldyen original spreadsheet documents.

pcardune committed:
Draft version of NPM-2

@aldyen, @pcardune and @jaycenhorton sat down and put this together from @aldyen original spreadsheet documents.

pcardune commented on NPM-2.md:
```diff
@@ -0,0 +1,132 @@
+energy use data the Supplier provides, except for the purposes the information
+provider directly approves.  The land management and energy use data that is
+used to establish the Project 10-year Projected CO2 Baseline will not be shared
+with Nori or stored in the Nori blockchain.
```
We plan to write up the categories of data that are required to be public or private or are not required at all. For example:

  * public essential

    * location of the fields

  * public nonessential

    * satelite imagery

  * private essential

    * fertilizers and nutrients

  * private nonessential

    * what the farmers like to do in their personal time

jaycenhorton replied:
Has been started in NMP-3

pcardune committed:
Adding another NMP document and associated NIP

Will come back later to rearrange these pieces into the proper number of NIPs/NMPs once it's more fleshed out.

paulgambill commented on NPM-2.md:
```diff
@@ -0,0 +1,132 @@
+term.  Provide a simplified explanation of the project/methodology being
+proposed.
+
+### Establishing Project/CRC Ownership & the "Responsible Person"
```
Why require a US-based person to act as agent? Is this a commodities regulatory requirement?

pcardune replied:
I'll let @Aldyen answer about the regulation part, but my understanding is that it reduces our risks as market operator because there will be a US resident subject to enforcement of US laws in the event of fraud or other nefarious activity. The other aspect is that this is standard practice for all commodities that are traded in US based markets, including things like oil and gas.

paulgambill commented on NPM-2.md:
```diff
@@ -0,0 +1,132 @@
+   
+  4. Tier 1 -- no uncertainty.
+   
+CRCs that derive from any Registered Project will be initially listed as
```
I would prefer that we call this a discount rate rather than listing them as "CRC1".

pcardune replied:
I find the word "discount" very confusing. They're not being offered at a lower price...?

paulgambill replied:
It's more like an "insurance" policy. I don't want the buyers seeing the CRCs rated at different tiers, because then they'll wonder why they're getting a "lower quality" CRC than some other buyer might get. The "discount rate" means that we hold back some of the tokens the supplier has earned in an escrowed smart contract that releases them over time as verification improves.

jaycenhorton replied:
We are both talking about the same end result of escrow. The user doesn't really see the crc1 vs discount, and calling it a crc of a certain tier is actually more just an internal distinction and only matters to the seller (in that the person who sold the crc is only going to get nori tokens at a certain rate) and never really presented to the buyer. So discount doesn't make sense if it's only a distinction to the seller. I think this is purely a technical jargon argument to keep it clear for us in implementation rather than a ux distinction.

Aldyen commented:
Jaycen has clearly ennunciated one of a few reasons I recommend CRC tiers
over discount.  But we haven't heard your detailed argument for
discounting, Paul 1.  Can we possibly set this question aside for now,
think about it, and put it on the agenda for our face-to-face meeting next
week? That would enable me to hear and consider your detailed argument for
discounts, and your plan for implementing the discounting method.

Tiering could be continuous--with CRC ratings from 1 to 100, based on
estimation uncertainties.  It doesn't have to be blocks.  So far, Paustian
prefers blocks. I could go either way.

Also, I would like to avoid the word "discount" in this context, even if we
ultimately settle on the detailed method you will propose. "Discou t" can
have negative conotatiins we do not intend.  And there are some approaches
to implementing a "discounting" method (whatever label we ultimately call
it) that would cause me more concern than others.

Some, but not all, of the primary design objectives that lead me to my
recommended CRC rating/restricted NORI method are:
--  all CRCs look the same to buyers, so buyers are not and do not need to
be conscious of CRC ratings/discounts (the ratings are just to communicate
to suppliers what their NORI token unrestricted/restricted splits will be);
--  any risk that we find we have allowed more CRCs to be listed than can
be justified when the CRCs are validated is born by suppliers and Nori, in
some combination, not CRC buyers (hence he need for the self-insure ncern
mechanisms that are embedded in the NMP) ;
--  CRCs with higher estimation error ratings convert into higher % of
restricted NORI tokens;
--  unrestricted NORI tokens are held in suppliers' wallets, so they can
record them as contingent revenues/assets on their balance sheets and
pledge them as security for their normal business activities (lenders will
apply their own discount to restricted NORI for purposes of determining
security value;  but lenders will also typically require, in lending
covenants, farmers to maintain regen practices as a condition of accepting
restricted NORI as security--that is very good for NORI, reducing our risk,
but not at Nori expense);
--  unrestricted NORI tokens are held in suppliers' wallets, so suppliers
and Nori have certainty that the NORI tokens will be there and available to
be released for unrestricted trade if/when the historical CR claims are
validated (if the NORI are not held in a trust account for the suppliers,
the suppliers have no guarantee that there will be any NORI available to
buy any future CRCs they might earn, if the plan is to approve only the
discounted CRC volume now and make up any difference by approving more CRCs
if future validation shows the stored more carbon than the historical
discounted CRC issuance suggests)

Aldyen

On Feb 23, 2018 10:22 AM, "Jaycen Horton" <notifications@github.com> wrote:

*@jaycenhorton* commented on this pull request.
------------------------------

In NPM-2.md
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170327418>:

> +
+  4. ... potentially more stuff ...
+
+Based on guidance from Nori, along with the information provided by
the Verifier,
+the administrator of the simulation tool will assign the Project Baseline to a
+"data quality tier".  The Project Baseline will be deemed one of:
+
+  1. Tier 4 -- high uncertainty
+
+  2. Tier 3 -- moderate uncertainty
+
+  3. Tier 2 -- low uncertainty.
+
+  4. Tier 1 -- no uncertainty.
+
+CRCs that derive from any Registered Project will be initially listed as

We are both talking about the same end result of escrow. The user doesn't
really see the crc1 vs discount, and calling it a crc of a certain tier is
actually more just an internal distinction and only matters to the seller
(in that the person who sold the crc is only going to get nori tokens at a
certain rate) and never really presented to the buyer. So discount doesn't
make sense if it's only a distinction to the seller. I think this is purely
a technical jargon argument to keep it clear for us in implementation
rather than a ux distinction.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170327418>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AhHgzg5clP0diMfnyxKl15gIOc5ClgaCks5tXwICgaJpZM4SP8zz>
.

Aldyen commented:
Paul,

ON US RESIDENT (or entity) AS "PERSON RESPONSBILE"

I recommend inclusion of this requirement, in part, to anticipate future
crypto regulatory action, but not entirely.  It is more about making the
CRC sales pitch more attractive to large corporate buyers.

In traditional security AND COMMODITY markets, a foreigner must have a US
agent in order to participate in the US markets

But this long-standing legal tradition isn't just for trading commodity
futures in the US.  Under long-standing US federal laws, a Canadian (or any
foreign) canola producer may not export canola into the US without first
setting up a US agent, which agent must be a corporation (often registered
in Delaware) or an officer of such a corporation (depending on the
context).  A foreign ethanol or biodiesel supplier must have a designated
US agent.  As must a foreign oil or natural gas supplier, or a foreign
pharmaceuticals, beer or pork supplier, or a foreign wood products supplier.

US law consistently requires foreign commodity suppliers to have a
designated US agent because it is through that agent that the US government
ensures that the foreign suppliers are in compliance with US reporting
rules that are primarily designed to protect US health and safety.

In commodity futures and securities markets, my Canadian broker's US
affiliate is technically my "broker", with respect to the shares I purchase
that trade over US exchanges, and the US affiliate is responsible for
compliance with US federal and state KYC rules with respect to my
investments and trades.

So we should anticipate that, as the regulators' approach to crypto
matures, a requirement that foreign crypto sellers and buyers must have a
US agent will be one of the first (we can hope only) reporting requirements
they will impose on crypto markets.

But I am not thinking only about insurance re: potential crypto regulation
in this regard.  Second, when US large emitters buyers and regulators look
hard at Nori, we want the CRC to look just like a ROC, in legal terms.

WHAT'S a "ROC"?

Existing US law allows biofuel producers to trade and bank "Renewable
Obligation Certificates", or "ROCs".  A ROC represents one US gallon of
biofuel.  When a (US or foreign) biofuel supplier registers in the US
(foreign  iofuel supplies have to register through a US corporation) and
complies with certain reporting requirements, the US EPA authorizes the
biofuel supplier to issue one ROC for each gallon of biofuel delivered into
the US.  (The EPA doesn't issue the ROCs.)  Then US refineries, blenders
and foreign finished product suppliers prove compliance with US renewable
fuel content mandate (the US Renewable Fuel Standard, or RFS) by remitting
ROCs to the US EPA. ROCs can be separated from the physical biofuel sale,
and they can be traded and banked.  So some refineries do not physically
comply with their US ethanol content mandates every year.  They buy surplus
ROCs from biofuel producers (who sell some of their fuel to users who are
not subject to the biofuel content mandates), electric utilities (who buy
biofuels, and can accumulate ROC inventories but are not be subject to a
biofuel mandate), or other refineries (whose fuel formulations exceed the
current minimum biofuel content mandate).

I want the CRC to look as much like a ROC as possible, not just when crypto
market regulators decide to look, but also when we are trying to convince
US electric utilities and refinery operators to buy CRCs to offset their
GHG emissions.

If our CRCs look like ROCs, it becomes easy to argue that by entering and
supporting the Nori market, among other things, the large emitters are
pre-empting future GHG regulation by proving/participating in a voluntary
CRC market that is as or more stringent/disciplined as the existing RFS/ROC
market, only more efficient and not requiring government bureaucracy.

Under the US RFS, a foreigner biofuel supplier cannot issue ROCs unless
they incorporate or select a US company as their agent and name a US
resident as "person responsible" for meeting any applicable US reporting
requirements.

So I think it is wise to include this requirement in the soils carbon NMP
because it anticipates the least intrusive and most likely element in
commodity regulation, makes thw CRC look like a ROC--making the pitch to US
CRC buyers much easier/attractive--and since it is already standard
practice for foreign food producers who sell any product into the US, it
shouldn't be an issue for most of the foreign CRC suppliers who want to
list on Nori..

It might be an issue for foreign food producers who sell nothing into the
US.  But that should be a small issue for them, since the global food
market infrastructure is already set up to help foreign food producers
efficiently meet with this requirement.

When it comes to other NMPs--say  for direct air capture--I am still
undecided whether or not to recommend the US agent requirement.  Leaving it
out increases the risk that those projects will not be permitted to list
CRCs on Nori, if they don't have a US agent, if/when the regulators
eventually rule US trade in CRCs generated by foreign entities must have
designated US representives.  But I don't imagine that any such CFTC ruling
will apply retroactively to CRCs.  (Not sure how they will with foreign
NORI token sellers in US exchanges, but that is not the topic of this
discussion.) Having said that, US agency should not be a problem for
Climeworks (given their relationship with Swiss Re) or CarbonEngineering
(head office in the US).

And leaving this requirement out of other methodologies would mean that not
all CRCs will look like ROCs.  I think all CRCs should look like ROCs,
especially to US large emitters who deal, daily, in ROCs.

Aldyen

On Feb 23, 2018 8:40 AM, "Paul Gambill" <notifications@github.com> wrote:

*@paulgambill* commented on this pull request.
------------------------------

In NPM-2.md
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170301972>:

> +
+  4. ... potentially more stuff ...
+
+Based on guidance from Nori, along with the information provided by
the Verifier,
+the administrator of the simulation tool will assign the Project Baseline to a
+"data quality tier".  The Project Baseline will be deemed one of:
+
+  1. Tier 4 -- high uncertainty
+
+  2. Tier 3 -- moderate uncertainty
+
+  3. Tier 2 -- low uncertainty.
+
+  4. Tier 1 -- no uncertainty.
+
+CRCs that derive from any Registered Project will be initially listed as

I would prefer that we call this a discount rate rather than listing them
as "CRC1".

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/nori-dot-eco/NMPs/pull/2#pullrequestreview-98973796>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AhHgzuHuZDRnOfCBRqRZ9TQvLiDUGEFvks5tXuoUgaJpZM4SP8zz>
.

Aldyen commented:
We are in full agreement in that regard. The buyers don't have to and won't
see the CRC ratings,  as we currently envision the methodology
implementation.  (I have said that all along, including in the email thread
with Keith Paustian.)

My understanding is that the Nori commitment to buyers is that through a
combination of fair but conservative year-to-year CRC quantification
methods and self-insurance, the buyers can be confident that every CRC they
buy = 1TCO2e  +/- 5% TO 10% (need Paustian to clarify what the smallest
post-validation estimation error will be).

We should be able to sell the proposed form of self-insurance to the
sellers--that we split any CRC1s that are created when validation shows
they stored more carbon than is reflected in the historical CRC listings.
That makes them participate in the creation of a CRC insurance pool  which
would cost them less (in $s and discomfort) than it would for us to apply
up-front absolute discount to to carbon removal claims in the year-to-year
CRC listing approval process.

Of course, this means that the CRC calculation method should be transparent
and fair, while it is conservative and open to community in our and
review.  COMET-Farm and Nori's joint commitments to open source
facilsoftware itate that,

Aldyen

On Feb 23, 2018 10:11 AM, "Paul Gambill" <notifications@github.com> wrote:

*@paulgambill* commented on this pull request.
------------------------------

In NPM-2.md
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170324623>:

> +
+  4. ... potentially more stuff ...
+
+Based on guidance from Nori, along with the information provided by
the Verifier,
+the administrator of the simulation tool will assign the Project Baseline to a
+"data quality tier".  The Project Baseline will be deemed one of:
+
+  1. Tier 4 -- high uncertainty
+
+  2. Tier 3 -- moderate uncertainty
+
+  3. Tier 2 -- low uncertainty.
+
+  4. Tier 1 -- no uncertainty.
+
+CRCs that derive from any Registered Project will be initially listed as

It's more like an "insurance" policy. I don't want the buyers seeing the
CRCs rated at different tiers, because then they'll wonder why they're
getting a "lower quality" CRC than some other buyer might get. The
"discount rate" means that we hold back some of the tokens the supplier has
earned in an escrowed smart contract that releases them over time as
verification improves.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170324623>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AhHgzrbvLtl9s3qU5jhR0hj8VIciNJiWks5tXv9SgaJpZM4SP8zz>
.

Aldyen commented:
I have been writing responses to the NMP comments while hiding g in the
bathroom.  I have to get back to my rehearsal.  Looking good forward to our
NMP ZOOM meeting go at 12:15 - 1 pm today.

Aldyen

On Feb 23, 2018 11:26 AM, "Aldyen Donnelly" <aldyen@nori.com> wrote:

We are in full agreement in that regard. The buyers don't have to and won't
see the CRC ratings,  as we currently envision the methodology
implementation.  (I have said that all along, including in the email thread
with Keith Paustian.)

My understanding is that the Nori commitment to buyers is that through a
combination of fair but conservative year-to-year CRC quantification
methods and self-insurance, the buyers can be confident that every CRC they
buy = 1TCO2e  +/- 5% TO 10% (need Paustian to clarify what the smallest
post-validation estimation error will be).

We should be able to sell the proposed form of self-insurance to the
sellers--that we split any CRC1s that are created when validation shows
they stored more carbon than is reflected in the historical CRC listings.
That makes them participate in the creation of a CRC insurance pool  which
would cost them less (in $s and discomfort) than it would for us to apply
up-front absolute discount to to carbon removal claims in the year-to-year
CRC listing approval process.

Of course, this means that the CRC calculation method should be transparent
and fair, while it is conservative and open to community in our and
review.  COMET-Farm and Nori's joint commitments to open source
facilsoftware itate that,

Aldyen

On Feb 23, 2018 10:11 AM, "Paul Gambill" <notifications@github.com> wrote:

*@paulgambill* commented on this pull request.
------------------------------

In NPM-2.md
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170324623>:

> +
+  4. ... potentially more stuff ...
+
+Based on guidance from Nori, along with the information provided by
the Verifier,
+the administrator of the simulation tool will assign the Project Baseline to a
+"data quality tier".  The Project Baseline will be deemed one of:
+
+  1. Tier 4 -- high uncertainty
+
+  2. Tier 3 -- moderate uncertainty
+
+  3. Tier 2 -- low uncertainty.
+
+  4. Tier 1 -- no uncertainty.
+
+CRCs that derive from any Registered Project will be initially listed as

It's more like an "insurance" policy. I don't want the buyers seeing the
CRCs rated at different tiers, because then they'll wonder why they're
getting a "lower quality" CRC than some other buyer might get. The
"discount rate" means that we hold back some of the tokens the supplier has
earned in an escrowed smart contract that releases them over time as
verification improves.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170324623>, or mute
the thread
<https://github.com/notifications/unsubscribe-auth/AhHgzrbvLtl9s3qU5jhR0hj8VIciNJiWks5tXv9SgaJpZM4SP8zz>
.

Aldyen commented:
I made a mistake in the prior message re: "US resident".  I should have
said "RIN", which stands for "Renewable Identification Number".  "ROC"
("Renewable Origin Certificate") is what the same certificate is called in
Europe.

Sorry,

Aldyen

On Fri, Feb 23, 2018 at 11:11 AM, Aldyen Donnelly <aldyen@nori.com> wrote:

> Paul,
>
> ON US RESIDENT (or entity) AS "PERSON RESPONSBILE"
>
> I recommend inclusion of this requirement, in part, to anticipate future
> crypto regulatory action, but not entirely.  It is more about making the
> CRC sales pitch more attractive to large corporate buyers.
>
> In traditional security AND COMMODITY markets, a foreigner must have a US
> agent in order to participate in the US markets
>
> But this long-standing legal tradition isn't just for trading commodity
> futures in the US.  Under long-standing US federal laws, a Canadian (or any
> foreign) canola producer may not export canola into the US without first
> setting up a US agent, which agent must be a corporation (often registered
> in Delaware) or an officer of such a corporation (depending on the
> context).  A foreign ethanol or biodiesel supplier must have a designated
> US agent.  As must a foreign oil or natural gas supplier, or a foreign
> pharmaceuticals, beer or pork supplier, or a foreign wood products supplier.
>
> US law consistently requires foreign commodity suppliers to have a
> designated US agent because it is through that agent that the US government
> ensures that the foreign suppliers are in compliance with US reporting
> rules that are primarily designed to protect US health and safety.
>
> In commodity futures and securities markets, my Canadian broker's US
> affiliate is technically my "broker", with respect to the shares I purchase
> that trade over US exchanges, and the US affiliate is responsible for
> compliance with US federal and state KYC rules with respect to my
> investments and trades.
>
> So we should anticipate that, as the regulators' approach to crypto
> matures, a requirement that foreign crypto sellers and buyers must have a
> US agent will be one of the first (we can hope only) reporting requirements
> they will impose on crypto markets.
>
> But I am not thinking only about insurance re: potential crypto regulation
> in this regard.  Second, when US large emitters buyers and regulators look
> hard at Nori, we want the CRC to look just like a ROC, in legal terms.
>
> WHAT'S a "ROC"?
>
> Existing US law allows biofuel producers to trade and bank "Renewable
> Obligation Certificates", or "ROCs".  A ROC represents one US gallon of
> biofuel.  When a (US or foreign) biofuel supplier registers in the US
> (foreign  iofuel supplies have to register through a US corporation) and
> complies with certain reporting requirements, the US EPA authorizes the
> biofuel supplier to issue one ROC for each gallon of biofuel delivered into
> the US.  (The EPA doesn't issue the ROCs.)  Then US refineries, blenders
> and foreign finished product suppliers prove compliance with US renewable
> fuel content mandate (the US Renewable Fuel Standard, or RFS) by remitting
> ROCs to the US EPA. ROCs can be separated from the physical biofuel sale,
> and they can be traded and banked.  So some refineries do not physically
> comply with their US ethanol content mandates every year.  They buy surplus
> ROCs from biofuel producers (who sell some of their fuel to users who are
> not subject to the biofuel content mandates), electric utilities (who buy
> biofuels, and can accumulate ROC inventories but are not be subject to a
> biofuel mandate), or other refineries (whose fuel formulations exceed the
> current minimum biofuel content mandate).
>
> I want the CRC to look as much like a ROC as possible, not just when
> crypto market regulators decide to look, but also when we are trying to
> convince US electric utilities and refinery operators to buy CRCs to offset
> their GHG emissions.
>
> If our CRCs look like ROCs, it becomes easy to argue that by entering and
> supporting the Nori market, among other things, the large emitters are
> pre-empting future GHG regulation by proving/participating in a voluntary
> CRC market that is as or more stringent/disciplined as the existing RFS/ROC
> market, only more efficient and not requiring government bureaucracy.
>
> Under the US RFS, a foreigner biofuel supplier cannot issue ROCs unless
> they incorporate or select a US company as their agent and name a US
> resident as "person responsible" for meeting any applicable US reporting
> requirements.
>
> So I think it is wise to include this requirement in the soils carbon NMP
> because it anticipates the least intrusive and most likely element in
> commodity regulation, makes thw CRC look like a ROC--making the pitch to US
> CRC buyers much easier/attractive--and since it is already standard
> practice for foreign food producers who sell any product into the US, it
> shouldn't be an issue for most of the foreign CRC suppliers who want to
> list on Nori..
>
> It might be an issue for foreign food producers who sell nothing into the
> US.  But that should be a small issue for them, since the global food
> market infrastructure is already set up to help foreign food producers
> efficiently meet with this requirement.
>
> When it comes to other NMPs--say  for direct air capture--I am still
> undecided whether or not to recommend the US agent requirement.  Leaving it
> out increases the risk that those projects will not be permitted to list
> CRCs on Nori, if they don't have a US agent, if/when the regulators
> eventually rule US trade in CRCs generated by foreign entities must have
> designated US representives.  But I don't imagine that any such CFTC ruling
> will apply retroactively to CRCs.  (Not sure how they will with foreign
> NORI token sellers in US exchanges, but that is not the topic of this
> discussion.) Having said that, US agency should not be a problem for
> Climeworks (given their relationship with Swiss Re) or CarbonEngineering
> (head office in the US).
>
> And leaving this requirement out of other methodologies would mean that
> not all CRCs will look like ROCs.  I think all CRCs should look like ROCs,
> especially to US large emitters who deal, daily, in ROCs.
>
> Aldyen
>
> On Feb 23, 2018 8:40 AM, "Paul Gambill" <notifications@github.com> wrote:
>
> *@paulgambill* commented on this pull request.
> ------------------------------
>
> In NPM-2.md
> <https://github.com/nori-dot-eco/NMPs/pull/2#discussion_r170301972>:
>
> > +
> +  4. ... potentially more stuff ...
> +
> +Based on guidance from Nori, along with the information provided by the Verifier,
> +the administrator of the simulation tool will assign the Project Baseline to a
> +"data quality tier".  The Project Baseline will be deemed one of:
> +
> +  1. Tier 4 -- high uncertainty
> +
> +  2. Tier 3 -- moderate uncertainty
> +
> +  3. Tier 2 -- low uncertainty.
> +
> +  4. Tier 1 -- no uncertainty.
> +
> +CRCs that derive from any Registered Project will be initially listed as
>
> I would prefer that we call this a discount rate rather than listing them
> as "CRC1".
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nori-dot-eco/NMPs/pull/2#pullrequestreview-98973796>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AhHgzuHuZDRnOfCBRqRZ9TQvLiDUGEFvks5tXuoUgaJpZM4SP8zz>
> .
>
>
>

pcardune committed:
Updated with more specific about settlement process

pcardune committed:
remove coderpad header

jaycenhorton committed:
Update NMP-3.md

Moved some to https://github.com/nori-dot-eco/NIPs/blob/NIP-6/NIP-6-Incremental-Nori-Release.md

jaycenhorton committed:
Delete NPM-2.md
